recipe/build.sh

```diff
@@ -45,6 +85,11 @@ cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
          -DWITH_UNIT_TESTS=OFF \
          -DINSTALL_MYSQLTESTDIR= \
          -DWITH_WSREP=OFF \
+         -DIMPORT_EXECUTABLES=../host/import_executables.cmake \
```

Comment:
Are we sure this is safe for non-m1 builds?

Reply:
Yes, if we don't cross compile, we just build the tools in advance..